tests/main_test.py

```diff
@@ -216,3 +217,15 @@ def test_result_as_dict():
                      'domain': 'google',
                      'suffix': 'com'}
     assert result._asdict() == expected_dict
+
+
+@responses.activate # pylint: disable=no-member
+def test_cache_timeouts():
+    server = 'http://some-server.com'
+    responses.add( # pylint: disable=no-member
+        responses.GET, # pylint: disable=no-member
```

Comment:
the test suite complained about `no-member`, and I'm not totally sure how to fix that properly. I saw this in a few other places in code, but if you can point me in the direction of how to do it right then I'd be happy to do so!

Reply:
🤷‍♂️ This seems fine. The alternative is [disabling linting of `responses` completely](https://github.com/getsentry/responses/issues/74#issuecomment-158025086).